modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/IgniteThrottlingUnitTest.java

```diff
@@ -140,17 +157,14 @@ public void testCorrectTimeToPark() {
 
         int markDirtySpeed = 34422;
         int cpWriteSpeed = 19416;
-        long time = throttle.getCleanPagesProtectionParkTime(0.04,
+        long time = throttle.getCleanPagesProtectionParkTime(0.67,
```

Comment:
The old code was too pessimistic when calculating the delays, the new code does it differently, so the results are different, hence the numbers in the test are changed. Also, 0.04 is too small to trigger throttling with the new code, so I had to change it to 0.67 to have ability to verify the non-zero duration.

As for the case when we don't need to throttle at all, there is `shouldNotThrottleWhenWritingSlowly()` that verifies exactly this. It uses dirty ratio 0.47, not 0.04, but for 0.04 it would be the same.